packages/react-ui-core/src/ToggleButton/ToggleButton.js

```diff
@@ -0,0 +1,66 @@
+import React, { Component } from 'react'
+import PropTypes from 'prop-types'
+import classnames from 'classnames'
+import themed from 'react-themed'
+import autobind from 'autobind-decorator'
+
+@themed(/^ToggleButton/)
+
+export default class ToggleButton extends Component {
+  static propTypes = {
+    className: PropTypes.string,
+    theme: PropTypes.object,
+    children: PropTypes.node,
+    value: PropTypes.bool,
+    onClick: PropTypes.func,
+  }
+
+  static defaultProps = {
+    theme: {},
+    value: false,
+  }
+
+  constructor(props) {
+    super(props)
+
+    this.state = {
+      value: props.value,
+    }
+  }
+
+  @autobind
+  toggle() {
+    const { onClick } = this.props
+    const value = !this.state.value
+    this.setState({
+      value,
+    })
+    if (onClick) onClick(value)
+  }
+
+  render() {
+    const {
+      className,
+      theme,
+      children,
+      onClick,
+      value,
+      ...props
+    } = this.props
+
+    return (
+      <button
```

Comment:
You should be using all react-ui-core components when possible i.e. `<Button/>`

Reply:
Changed to Button